scripts/babel/__tests__/transform-test-gate-pragma-test.js

```diff
@@ -28,6 +30,23 @@ describe('transform-test-gate-pragma', () => {
     });
   };
 
+  // eslint-disable-next-line no-unused-vars
+  const _test_gate_semver_for_devtools = (range, testName, cb) => {
+    test(testName, (...args) => {
+      shouldPass = semver.satisfies('18.0.0', range);
```

Comment:
~~I don't know why "18.0.0" is hardcoded here. Shouldn't this value come in via a `@gate react_version` comment rather than be hard-coded?~~ I don't think I understand the purpose of these tests :D